packages/itwinui-react/src/core/utils/hooks/useOverflow.tsx

```diff
@@ -78,7 +78,7 @@ export const useOverflow = <T extends HTMLElement>(
     if (availableSize < requiredSize) {
       const avgItemSize = requiredSize / visibleCount;
       const visibleItems = Math.floor(availableSize / avgItemSize);
-      setVisibleCount(visibleItems);
+      setVisibleCount(visibleItems > 0 ? visibleItems : 1);
```

Comment:
maybe add a comment above this so we remember this bug when we open this file again in 2 years